using/user-administration.md

```diff
@@ -91,4 +116,62 @@ or delete the user's access to the environment entirely.
 Add a new user by clicking on the `Add user` button.
 
 > **note**
-> After a new user is added to the environment, it needs to be rebuilt (git push) for the changes to take effect. When the environment rebuild is complete, allow a minute for the routes to update fully and for the new user to be able to use SSH access.
+> Remember the user will only be able to access the environment once it has 
+> been rebuilt (after a `git push`)
+
+------------------------------------------------------------------------
+## Manage user with the CLI
+You can user the Platform.sh command line client to fully manage your users
+and integrate this with any other automated system.
+
+Available commands:
+
+* user:add
+  * Add a user to the project
+* user:delete
+  * Delete a user
+* user:list (users) 
+  * List project users
+* user:role
+  * View or change a user's role
+
+For example the following command would add the admin to alice@example.com role to the current 
+project.
+
+```
+   platform user:add 
+```
+
+Will present you with an interactive wizard that will allow you to choose
+precisely what rights you want to give the new user.
+
+````bash
+$ platform user:add
+
+Email address: alice@example.com
+The user's project role can be 'viewer' ('v') or 'admin' ('a').
+Project role [V/a]:
+The user's environment-level roles can be 'viewer', 'contributor', or 'admin'.
+development environment role [V/c/a]:
+sprint1 environment role [V/c/a]:
+hot-fix environment role [V/c/a]:
+master environment role [V/c/a]:
+pr-2 environment role [V/c/a]:
+pr-3 environment role [V/c/a]:
+Summary:
+    Email address: alice@example.com
+    Project role: viewer
+    development: viewer
+    sprint1: viewer
+    hot-fix: viewer
+    pr-2: viewer
+    pr-3: viewer
+Adding users can result in additional charges.
+Are you sure you want to add this user? [Y/n]
+User alice@example.com created
+```
+
+Once this has been done the user will receive an email asking her to confirm
```

Comment:
Agreed, I generally prefer they/them as neutral. Almost never "he/him" (you won't find any in the docs afaik). In the example we have Alice. I believe here she is fine.
But feel free to amend